### ISSUE TITLE
Sanitizing a property is not limited to class where it is applied

### ISSUE THREAD

**Issue by aljazerzen** [typestack/class-sanitizer#10]
When a decorator is applied to a field it is applied not only when validating instances of its class, but when validating any class. 

Consider this:
```typescript
export class A {
  message: string;
}

export class B {
  @Escape()
  message: string;
}

let a = new A();
a.message = '<';

sanitize(a);
console.log(a);

// Expected output: A { message: '<' }
// Actual output: A { message: '&lt;' }
```

I suspect that [MetadataStorage.getSanitizeMetadatasForObject()](https://github.com/typestack/class-sanitizer/blob/master/src/metadata/MetadataStorage.ts#L59-L69) is not working correctly.

**Comment by Kanti** [comment]
hi @pleerock @YouriT,

are you still using class sanitizer or?
We would like to get this bug fixed.

Or should we just use https://github.com/hollowverse/class-sanitizer

**Comment by YouriT** [comment]
@Kanti yes still using it. Indeed it looks like unmaintained at this point. I personally don't have the resources to dedicate maintenance to the project.

Maybe @forabi wants to maintain the project? If so, @pleerock could kick in the discussion to avoid this project dying.

**Comment by YouriT** [comment]
addressed in https://github.com/swanest/class-sanitizer/commit/a31bdb903f6e27b66cd3b58f60d25bc8ce1ae125

**Comment by lucasmonstrox** [comment]
News? Im having this issue :(

**Comment by NoNameProvided** [comment]
Fixed in v1.0.0.
